csrc/scheduler/utils.h

```diff
@@ -880,6 +880,35 @@ void buildAllocationDomainFromLoopIds(TensorView* tv);
 // For shared memory tensor, replay loop domain transformations to allocation
 // domain
 void buildAllocationDomainForSharedMemoryTvs(Fusion* fusion);
+
+// TMA pointwise scheduler uses 2D tensor domain and 2D TMA tile.
```

Comment:
I'm still having a trouble to follow the comment below. My memory of how TMA is lowered in nvFuser is almost gone already. Please give an explanation without assuming prior knowledge.

Reply:
maybe this doc can help? https://github.com/NVIDIA/Fuser/blob/main/doc/dev/tma.md

Reply:
Yes, please use it as a reference but try to make this description as self contained as possible. I suppose functions like this would eventually become common building blocks for TMA+WS scheduling, so it's important to make it quickly understandable.